### PR TITLE
feat(category): move page actions to "DaffCategoryPageActions" and "DaffCategoryPageActionTypes"

### PR DESCRIPTION
**Pull request by damienwebdev**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[x] Yes
[ ] No
```

The type has changed, users will have to switch to the new type.

## Other information

### CODE REVIEW

**Review by griest024 — APPROVED** [review]

**Comment by damienwebdev** [comment]
needs @lderrickable review

**Review by lderrickable — CHANGES_REQUESTED** [review]
lgtm. Waiting on @damienwebdev 's approval of the eslint PR to avoid conflicts.

**Comment by damienwebdev** [comment]
@lderrickable rebased.

**Review by lderrickable — APPROVED** [review]

**Pull request merged**
